config/initializers/assets.rb

```diff
@@ -49,7 +48,9 @@
                   hidden_values
                   password_strength
                   proxy_status
-                  parameter_override)
+                  parameter_override
+                  statistics
```

Comment:
That is correct. The file was removed when I chose to add inline js to the erb file as explained above. I corrected the assets.rb.